src/@next/components/organisms/AddressForm/AddressFormContent.tsx

```diff
@@ -0,0 +1,130 @@
+import React, { useCallback } from "react";
+
+import { TextField } from "@components/molecules";
+import { maybe } from "@utils/tsUtils";
+
+import * as S from "./styles";
+import { PropsWithFormik } from "./types";
+
+export const AddressFormContent: React.FC<PropsWithFormik> = ({
+  formRef,
+  handleChange,
+  handleBlur,
+  errors,
+  handleSubmit,
+  values,
+}) => {
+  const basicInputProps = useCallback(
+    () => ({ onBlur: handleBlur, onChange: handleChange }),
+    [handleChange, handleBlur]
+  );
+
+  return (
+    <S.AddressForm ref={formRef} onSubmit={handleSubmit}>
+      <S.Wrapper>
+        <S.RowWithTwoCells>
+          <TextField
+            name="firstName"
+            label="First Name"
+            value={values!.firstName || ""}
```

Comment:
`values` should be defined with all fields set to empty ("") as default, unless provided with data from api. Just `values.firstName` should be enough